strip-coverletter-docker.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu
+set -xv
+in=$(realpath $1)
+bname_in=${in##*/} # /foo/bar/baz.pdf = baz.pdf
+
+out=$2
+bname_out=${out##*/}
+
+mkdir -p vol
+cp $in vol/$bname_in
```

Comment:
you may not need to copy the input file: mount also `--volume $in:/data/$bname_in` after the other volume (the `/data` folder)